crawler.py

```diff
@@ -260,7 +261,8 @@ def run_all():
 
                 from lcov_rewriter import LcovFileRewriter
 
-                jsvm_output_file = 'jsvm_lcov_output.info'
+                os.makedirs('jsvm_output', exist_ok=True)
```

Comment:
One last suggestion, if you put this directory inside jsvm_dir, you won't have to manually remove it.
```
jsvm_output_dir = os.path.join(jsvm_dir, 'jsvm_output')
jsvm_output_file = os.path.join(jsvm_output_dir, 'jsvm_lcov_output.info')
```